src/main/resources/application.yml

```
@@ -8,7 +8,7 @@ management:
 
 testing-support.enabled: ${TESTING_SUPPORT_ENABLED:}
 
-jwtKey: ${JWTKEY:wThK0f0/lh3FlxFcL4xUWDMI5C1J9KyQBgXV4wseh1e5J1uYJIjvTvArHxQDrYoHJ23xFxjHkOnvNbR5dXRoxA==}
+jwtKey: ${JWTKEY:}
```

probably remove this, or at least do:
```suggestion
jwtKey:
```